vm/devices/storage/disk_layered/src/lib.rs

```diff
@@ -652,9 +654,11 @@ impl DiskIo for LayeredDisk {
         buffers: &RequestBuffers<'_>,
         sector: u64,
     ) -> Result<(), DiskError> {
+        let mut bounce_buffers = None::<(OwnedRequestBuffers, GuestMemory)>;
```

Comment:
am I reading this right? we're not reusing these bounce-buffers between ops?

Reply:
We are not. We are relying on malloc to be good.

Reply:
Ehh. If it's a perf limiting factor, we'll discover that via trivial perf analysis.

Besides, we don't currently have a server/cloud scenario for layered disks. The only place this will matter is at server scale.

Reply:
that's a fair point. alrighty